docs/development/release.rst

```diff
@@ -120,7 +162,7 @@ Make a Bugfix release
    `gammapy-webpage repo <https://github.com/gammapy/gammapy-webpage>`__. The ``datasets`` entry should point to last
    stable version, like ``v1.0`` or ``v1.1``. We do not provide bug-fix release for data.
 
-#. Follow the  `Astropy bug fix release instructions <https://docs.astropy.org/en/latest/development/maintainers/releasing.html#maintaining-bug-fix-releases>`__.
+#. Follow the `Astropy bug fix release instructions <https://docs.astropy.org/en/latest/development/maintainers/releasing.html#maintaining-bug-fix-releases>`__.
```

Comment:
Probably we could expand here as well.

Reply:
Yes I agree. Do we know the exact steps we take, or do we need to wait until we do a bug release?

Reply:
We can wait for now